docs/api_endpoints.md

```diff
@@ -0,0 +1,172 @@
+# API Endpoints provided by marc_liberation / Bibdata
+
+This documentation does not include all API endpoints. See [config/routes.rb](../config/routes.rb) for more endpoints.
+
+## POST '/bibliographic/:mms_id/holdings/:holding_id/items/:item_pid/discharge' (item
+discharge)
+
```

Comment:
@hackartisan looks great. Can you please adjust the (item discharge ) comment so that it has the same format?

Reply:
done, thanks